xcode/Subconscious/Shared/AppTheme.swift

```diff
@@ -57,26 +57,26 @@ extension UIFont {
 }
 
 //  MARK: Color
-//  String color names are references to ColorSet assets, and can be found
-//  in Assets.xassets. Each ColorSet contains a light and dark mode, and color
-//  is resolved at runtime.
-//  2021-12-15 Gordon Brander
 extension Color {
     static let separator = SwiftUI.Color(uiColor: UIColor.separator)
-    static let text = SwiftUI.Color("TextColor")
-    static let placeholderText = SwiftUI.Color("PlaceholderTextColor")
-    static let textPressed = text.opacity(0.5)
-    static let textDisabled = placeholderText
-    static let tertiaryText = placeholderText
+    static let placeholderText = SwiftUI.Color(
+        uiColor: UIColor.placeholderText
+    )
+    static let disabled = placeholderText
+    static let tertiaryLabel = SwiftUI.Color(uiColor: .tertiaryLabel)
```

Comment:
UIKit color naming convention is inconsistent, but I think it's better to stay in keeping with the UIKit color name here.